packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -1,27 +1,185 @@
 import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
-// import { SliderState } from './Slider.types';
+import { HighContrastSelector } from '@fluentui/style-utilities';
+import { createFocusIndicatorStyleRule } from '@fluentui/react-tabster';
+import { SliderState } from './Slider.types';
 
 /**
  * Styles for the root slot
  */
-const useStyles = makeStyles({
+const useRootStyles = makeStyles({
   root: theme => ({
-    // TODO Add default styles for the root element
+    '--slider-thumb-size': '20px',
```

Comment:
We have not been doing raw css variables like these for other converged components I think. I wonder what everyone's opinions are on this as I know there are some downsides like losing types and such.

Reply:
I think none of css i really typed, and that it should be an anti-practice to hardcode values that are repeated and potentially overridden. If not a variable, you could use a const, but it creates new bugs. E.g. I believe down below, slider-thumb-size is used as the height of the container, as well as a few other places (calc'd on the left/right on the rail). If you change the variable, all those other places will correctly adjust. If you don't use a variable, you end up with lots of other things to fix.

Reply:
Agreed, this seems like something that should probably be discussed rather than just adding a new convention which may not be followed elsewhere